requirements.txt

```diff
@@ -9,3 +9,4 @@ jsonschema
 pyzmq>=16.0.2
 broadbean>=0.9.1
 wrapt
+pandas
```

Comment:
Should pandas be added to the real requirements in setup.py? Alternatively could we move the pandas import into the functions that use it so that it's only a dependency when used

Reply:
Yeah, I was also unsure about this one. @WilliamHPNielsen convinced me to have it as a general requirement.
When moving the import to the function we would also need to fix the type annotations of pandas objects used outside, or in in the header for the function...